src/app/DTT/upload/Image.js

```diff
@@ -100,12 +120,77 @@ const Read = ({ code, data, parentCode, variant, config }) => {
         })
       : null
 
+  const theme = useTheme()
+  const { buttonBackgroundColor } = useProductColors()
+
   const bg = useColorModeValue('gray.300', 'gray.600')
   if (variant === 'profile_image') {
     return <Image {...config} src={src} alt="profile-picture" w="10rem" borderRadius="xl" />
   }
 
-  if (!!carddisplay) {
+  const imagePreviewCount = min(srcList.length, 3)
+
+  const getMultiImageStyling = index => {
+    return {
+      roundedBottomLeft: index === 0 ? 25 : 0,
+      roundedTopLeft: index === 0 ? 25 : 0,
+      roundedBottomRight: index === imagePreviewCount - 1 ? 25 : 0,
+      roundedTopRight: index === imagePreviewCount - 1 ? 25 : 0,
+    }
+  }
+
+  if (multiUpload) {
+    return (
+      <Box width={`${(100 / 3) * imagePreviewCount}%`}>
+        <Modal isOpen={isOpen} onClose={onClose} size={'4xl'}>
+          <ModalOverlay />
+          <ModalContent>
+            <ModalHeader>Images</ModalHeader>
+            <ModalCloseButton />
+            <ModalBody>
+              <MultiImageViewer uuidList={safelyParseJson(data?.value)} />
+            </ModalBody>
+          </ModalContent>
+        </Modal>
+
+        <Box width={'100%'} position={'relative'}>
+          <Button
+            onClick={onOpen}
+            zIndex={5}
+            position={'absolute'}
+            right={'8px'}
+            bottom={'8px'}
+            rounded={100}
+            backgroundColor={theme.colors.background.light}
+            color={buttonBackgroundColor}
+            _hover={{
+              background: buttonBackgroundColor,
+              color: theme.colors.background.light,
+            }}
+          >
+            {`View ${srcList.length} photos`}
+            <Box mr={2} />
+            <FontAwesomeIcon icon={faImages} />
+          </Button>
+          <HStack justifyItems={'flex-start'} zIndex={1}>
+            {srcList.slice(0, imagePreviewCount).map((value, index) => (
+              <Image
+                maxH={'250px'}
+                fit={'cover'}
+                width={`${100 / imagePreviewCount}%`}
```

Comment:
Can we remove the width and height from img tag here? We can use AspectRatio from chakra ui to set the height and width of the container holding the image.

Reply:
huh ok, I'm not familiar with the aspect ratio widget, I'll look into it

Reply:
Ok, updated to reflect your suggestions as well